go.mod

```diff
@@ -1,15 +1,13 @@
 module github.com/bitcoin-sv/go-paymail
 
-go 1.21.5
+go 1.22
+
+toolchain go1.22.5
```

Comment:
Why the `toolchain` appeared?
- In `spv-wallet` for example, we have only `go <version>`
```
module github.com/bitcoin-sv/spv-wallet

go 1.22.5
```

Reply:
🤷  sorry, didn't see this change when took this draft PR moved to 1.22.5